docs/trace_viewer.md

```diff
@@ -148,11 +158,3 @@ lines may or may not appear in certain profiles.
     via a combination of user annotations, built-in heuristics, and by
     post-processing information it receives from different components (e.g.,
     CUPTI drivers, kernel launch IDs, TPU runtime information, etc.).
-*   Clicking on an XLA op provides additional information in the details pane.
-    For example, it links to the op in the Graph Viewer tool. It may also
-    provide pointers to source code and/or the Python stack trace, the framework
-    op that caused this XLA op to get generated, etc. (if present in the
-    profile). It may also show FLOPS (number of floating point operations
-    executed by the op) and bytes accessed by the op; this information is
-    statically acquired from XLA during compilation, rather than runtime
-    information from the profile.
```

Comment:
I reorganized this content to be included in the details pane description as I felt it would be more self-contained there.